### PR TITLE
Translators.md update

### PR DESCRIPTION
**Pull request by thisnamewasnottaken**
Updated the translators.md file and linked to the wiki. Added all the workflow steps for different levels of contributors.

### CODE REVIEW

**Comment by elstoc** [comment]
I'm not sure I'm qualified to review the amount of changes here - I don't know anything about the translation process. I would suggest someone else review it for accuracy (and merge or not) and then I'll submit a copy-edit PR afterwards if required.

**Comment by thisnamewasnottaken** [comment]
Thanks! Happy to take any comments/ suggestions on board.

File: doc/TRANSLATORS.md [hunk 1]
```diff
@@ -1,82 +1,113 @@
...
+6. Start translating. Saving will update `<lang>.po`.
 
-7. When you are done, `bzip2` your PO file and send it to the mailing list.
+7. When you are done, `bzip2` your PO file and send it to the [mailing list](https://www.darktable.org/contact/) or per instructions on the [wiki](https://github.com/darktable-org/darktable/wiki/Translations).
 
-Now about updates. Every new version of darktable contains new functionality and thus there are always new messages to translate. It's best to keep track of changes, but even so we announce strings freeze a few days before releasing. It means that no changes to translatable messages will be added, so your translation created during that period will work fine.
+8. That's all!
 
-To update your translation from working your tree:
+## Translate the darktable User Interface for darktable releases (how-to, git)
```
Comment (TurboGit) [inline doc/TRANSLATORS.md:49]: For darktable releases we use the committed pot file as described above. I would completely remove this section. At this stage translators should not care about Git at all.

**Review by TurboGit — CHANGES_REQUESTED** [review]
Some more clean-up needed.

File: doc/TRANSLATORS.md (hunk 1 above)
Comment (thisnamewasnottaken) [reply doc/TRANSLATORS.md]: Made changes as requested.

**Review by TurboGit — APPROVED** [review]
Looks better to me. Thanks!

**Pull request merged**